Angular/src/app/fragments/fragments.component.ts

```diff
@@ -223,4 +225,21 @@ export class FragmentsComponent implements OnInit, OnDestroy {
       return 'Show translation';
     }
   }
+
+  /**
+   * Opens a dialog to set a custom filter. If filter set, requests documents from server
+   * @param number of column_id to load documents into
+   * @author Ycreak
+   */
+  protected set_custom_filter(column_id: number): void {
+    const dialogRef = this.matdialog.open(DocumentFilterComponent, {});
+    dialogRef.afterClosed().subscribe({
+      next: (document_filter) => {
+        if (document_filter) {
+          console.log(document_filter, 'data');
```

Comment:
Console.log die je er mss nog uit wilt halen

Reply:
leaving console.logs in the code is one of my character flaws.